packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -424,6 +424,63 @@ function getTextFromInnerText(domSource: ALDOMTextSource, source: ALElementText[
   return null;
 }
 
+// Some environments (like JEST) may not have CSS
+export const cssEscape = typeof window.CSS?.escape === 'function'
+  ? (str: string) => window.CSS.escape(str)
+  : (str: string) => str.replace(/['"\[\]\(\)]/g, m => {
+    return "\\" + m;
+  });
+
+//https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#labelable
+const LabelableElememts = /BUTTON|INPUT|METER|OUTPUT|PROGRESS|SELECT|TEXTAREA/;
+function getTextFromElementLabel(domSource: ALDOMTextSource, source: 'label', results: ALElementText[]): ALElementText[] | null {
+  const { element, surface } = domSource;
+
+  // Many labelable elements could have a label assigned to them a few possible ways
+  const labels = (element as { readonly labels?: NodeListOf<HTMLLabelElement>; }).labels;
```

Comment:
This cast just removes the need for the `instanceof HTMLInputElement` check?

Reply:
Yes, although this is just a compile time check. But, I checked all the places `.labels` defined in the spec and all of them for sure are of the right type. But, since we are applying it to generic HTMLElement nodes, we have to make it explicit. 

Besides, we don't depend on the nodes being label elements since we will use their innerText anyways. So, it is safe to do compile time check only.